macros/general/troubleshooting-ticket.mdx

```diff
@@ -0,0 +1,29 @@
+---
+macro: troubleshooting-ticket
+---
+
+If you did not manage to identify the error and solve it by yourself, [open a support ticket](/account/how-to/open-a-support-tickets/#writing-an-effective-subject-and-description), and provide as many details as possible, along with the necessary information below:
+    - Resource ID
+    - Resource name
+    - Request type (PUT, GET, etc.)
+    - HTTP status code
+    - Date and time (timestamp)
+    - User-agent (SDK, client, console, etc.)
+    - Transaction ID (if possible)
+    - Log / trace of the error (if possible)
+    - Screenshots
```

Comment:
Could we link to the tool they should use to provide screenshots? Maybe check with Support the service they generally ask clients to use (I can't remember it off the top of my head)